secret-word/src/App.js

```diff
@@ -8,9 +8,9 @@ import { useCallback, useEffect, useState } from "react";
 import { wordsList } from "./data/words";
 
 // Components
-import StartScreen from "./components/StartScreen";
-import GameOver from "./components/GameOver";
-import Game from "./components/Game";
+import StartScreen from "./components/StartScreen/StartScreen";
```

Comment:
Para evitar essa repetição de nome no import, podemos utilizar um arquivo index.js dentro de cada pasta (Game, GameOver e StartScreen)

Nesse index conterá o seguinte código:
`export * from './StartScreen'`

Quando importamos algo de uma pasta, automaticamente ele procura o index, então deixa de ser necessário importar o arquivo e a pasta já que ela está sendo exportada no index.

Reply:
É uma boa estratégia mesmo!
Estou tentando aqui fazer esse ajuste, porém quando faço isso para o componente StartScreen, por exemplo, após fazer o import no App.js da seguinte forma: 
`import StartScreen from "./components/StartScreen/";` 

Ocorre o seguinte erro:
`export 'default' (imported as 'StartScreen') was not found in './components/StartScreen/' (module has no exports)`

Isso acontece para os outros componentes também. Estou pesquisando aqui para ver como resolver.

Reply:
Boa Matheus!! Acredito que já tenha resolvido, mas se persistir me avisa

Reply:
Ajustes feitos. 
:D